apps/spectator/components/game/Panel/Panel.css.ts

```diff
@@ -1,27 +1,34 @@
-import { theme } from '@hcc/styles';
-import { styleVariants } from '@vanilla-extract/css';
+import { rem, theme } from '@hcc/styles';
+import { style, styleVariants } from '@vanilla-extract/css';
+
+const panelItemBase = style({
+  ...theme.textVariants.default,
+  textAlign: 'center',
+  padding: `${rem(16)} 0`,
+  color: theme.colors.gray[4],
+  borderTop: `solid ${theme.colors.gray[2]}`,
+  borderBottom: `solid ${theme.colors.gray[2]}`,
+  borderWidth: 1,
+});
 
 export const panel = styleVariants({
   wrapper: {
     position: 'relative',
-    borderRadius: '0.75rem',
-    borderWidth: '2px',
-    borderColor: theme.colors.gray[2],
   },
   menu: {
     display: 'grid',
-    gridTemplateColumns: 'repeat(4, minmax(0, 1fr))',
-    borderRadius: '0.5rem',
+    gridTemplateColumns: 'repeat(3, minmax(0, 1fr))',
     width: '100%',
   },
-  item: {
-    borderRadius: '0.75rem',
-    padding: '0.75rem 0',
-  },
-  itemSelected: {
-    borderRadius: '0.75rem',
-    padding: '0.75rem 0',
-    background: theme.colors.secondary[2],
-    color: theme.colors.primary[3],
-  },
+  item: [panelItemBase],
+  itemSelected: [
+    panelItemBase,
+    {
+      background: theme.colors.secondary[2],
+      color: theme.colors.primary[3],
+      borderTop: 0,
+      borderBottom: `solid ${theme.colors.primary[3]}`,
+      borderWidth: 0.5,
```

Comment:
```suggestion
      borderBlock: `0.5px solid ${theme.colors.gray[2]}`,
```

'0.5px' 이게 적용되는지는 잘 모르겠긴 한데, 안되면 저것만 따로 빼거나 문자열 리터럴을 사용해도 될 것 같네요!